src/Raven.Server/Routing/RequestRouter.cs

```diff
@@ -310,6 +312,8 @@ await using (var writer = new AsyncBlittableJsonTextWriter(ctx, context.Response
                             _lastRequestTimeUpdated = now;
                         }
 
+                        _ravenServer.Statistics.UpdateLastCertificateRequestTime(certificateThumbprint, now);
```

Comment:
This is on the routing level, maybe we just want to track connection time. It might have less perf impact if we track it on the connection level, no?

Reply:
After talking with @gregolsky we decided to keep current implementation, there is a throttling on update implemented already and moving that to 'OnConnectionAsync' would cause very unstable results, because connections might be opened for a very long time and we would like to avoid getting wrong results in the triggers